### ISSUE TITLE
Feature suggestion: Show value on mouse hover and serial monitor below lifegraph 

### ISSUE THREAD

**Issue by ElfyShort** [chillibasket/processing-grapher#41]
Thank you for this amazing software.
It is very easy to setup and use.
However, in my case I am missing these 2 features to be able to fully migrate my project to this software:
1. I would like to have live data running in a panel below the life graph, as I need to record a screen, capturing both the graph and the raw data. Now as they split in tabs it is impossible to capture both.
2. To be able to compare how close one signal is to another at particular point in time. 
A suggestion is to show a ruler on mouse hover, and show all values next to it.
Basically, the same feature exists in SerialPlot:
![280271777-5e481c00-7077-4476-9928-a7a987e5e4b9](https://github.com/chillibasket/processing-grapher/assets/52491713/9e994dc0-d274-408c-bdb8-2f14bffb5ffb)
Thank you for consideration.
